runtime/Makefile

```diff
@@ -70,12 +70,15 @@ third-party-pkgs:
 ifneq (, $(THIRD_PARTY_PKGS))
 # Qthreads may use hwloc and/or jemalloc. Ensure they are built first
 ifneq (, $(filter $(THIRD_PARTY_PKGS),hwloc))
-	$(MAKE) -C $(THIRD_PARTY_DIR) hwloc
+	@echo "Making hwloc..."
+	@$(MAKE) -C $(THIRD_PARTY_DIR) hwloc
 endif
 ifneq (, $(filter $(THIRD_PARTY_PKGS),jemalloc))
-	$(MAKE) -C $(THIRD_PARTY_DIR) jemalloc
+	@echo "Making jemalloc..."
+	@$(MAKE) -C $(THIRD_PARTY_DIR) jemalloc
 endif
-	$(MAKE) -C $(THIRD_PARTY_DIR) $(THIRD_PARTY_PKGS)
+	@echo "Making third-party-packages..."
```

Comment:
Seems a little odd to call out hwloc/jemalloc specifically but not the others -- can we just put "Making third party packages" before any of them are built?

Reply:
That's just to make the authors of these lines feel bad for not adding the ordering dependencies in a more principled way.  :)

Seriously, though, it's to highlight that the repetition is intentional (i.e., that third-party-pkgs isn't returning a duplicate).